src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.cs

```diff
@@ -306,6 +307,59 @@ public async Task When_IsEnabled_Set()
 			Assert.AreEqual(contentPresenter.Foreground, foregroundColor);
 		}
 
+#if __ANDROID__
+		[TestMethod]
+		public async Task When_Text_IsWrapping_Set()
+		{
+			var textbox = new TextBox();
+
+			var initialHeight = 500;					
+			
+			textbox.Height = initialHeight;
+			
+			WindowHelper.WindowContent = textbox;
+
+			await WindowHelper.WaitForLoaded(textbox);
+
+			var actualHeigth = textbox.ActualHeight;
+
+			textbox.Text = "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremaaaaaaaaaaaaaaaaaa";
+			textbox.TextWrapping = TextWrapping.Wrap;
+			await WindowHelper.WaitForIdle();
+
+			var textBoxView = textbox.FindFirstChild<TextBoxView>();
+
+			Assert.IsTrue(textBoxView.LineCount > 1);
+		}
+
+
+		[TestMethod]
+		public async Task When_Text_IsNoWrap_Set()
+		{
+			var textbox = new TextBox();
+
+			var initialHeight = 250;
+
+			textbox.Height = initialHeight;
+
+			var oldActualH = textbox.ActualHeight;
+		
+			WindowHelper.WindowContent = textbox;
+
+			await WindowHelper.WaitForLoaded(textbox);
+
+			textbox.Text = "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremaaaaaaaaaaaaaaaaaa";
+			textbox.TextWrapping = TextWrapping.NoWrap;
+		
+			await WindowHelper.WaitForIdle();
+
+			var textBoxView = textbox.FindFirstChild<TextBoxView>();
+
+			Assert.IsTrue(textBoxView.LineCount <= 1);
```

Comment:
I'm confused by `<= 1`
Shouldn't it be just `== 1`? If not, it might be worth explaining that in a comment.